docs/plugins/mdx-pages/gatsby-node.js

```diff
@@ -0,0 +1,88 @@
+const componentWithMDXScope = require('gatsby-mdx/component-with-mdx-scope')
+
+const createMDXPagePath = node =>
+  `/${node.parent.name}`
+
+// Relate an MDX node with the page that we create below.
+//
+// Tried to create these nodes inside the createPages callback,
+// but they don't have an effect unless we also create a node field here.
+// for some reason.
+// not sure.
+const onCreateNode = ({ node, actions, getNode }) => {
+  const { createNodeField } = actions
+
+  if (node.internal.type !== 'Mdx') {
```

Comment:
Oh `Mdx`? Not `mdx`?

Reply:
this is the node type generated by gatsby-mdx. it's referenced here by them as well

https://github.com/ChristopherBiscardi/gatsby-mdx/blob/master/packages/gatsby-mdx/gatsby/extend-node-type.js#L34